descriptors/ModuleDescriptor-template.json

```diff
@@ -472,6 +481,11 @@
       "displayName": "Change Manager - create jobExecutions",
       "description": "Post JobExecution"
     },
+    {
+      "permissionName": "change-manager.jobexecutions.delete",
+      "displayName": "Change Manager - delete jobExecutions",
+      "description": "Delete JobExecutions Using JobExecutionIds"
```

Comment:
nit: Thinking this could be just "Delete jobExecutions". If you want to demonstrate that multiple ids can be accepted, you could include the word "batch" so maybe "Batch Delete JobExecutions"

Reply:
Hi @okolawole-ebsco - Updated changes as per review comment, Thanks 🙂